pkg/volume/glusterfs/glusterfs.go

```diff
@@ -346,7 +346,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 			return nil
 		}
 
-		const invalidOption = "Invalid option auto_unmount"
+		const invalidOption = "Invalid"
```

Comment:
`Invalid argument`

Reply:
It was originally Invalid argument but some versions exit with Invalid option auto_mount, so setting only Invalid should match both -> https://github.com/kubernetes/kubernetes/issues/50463#issuecomment-322030541

Reply:
Should it be `Contains(err, "Invlaid option") || Contains(err, "Invalid argument")` then?

Reply:
Yeah, looks like 'return' differs in different setups. we need to accommodate both, "invalid option" or "invalid argument"  or in worst case we should stick with just `invalid`.

Reply:
please check both "invalid option" and "invalid argument". If still failing, log the output.